example/app.js

```diff
@@ -13,50 +15,64 @@ app.use(function(req, res, next) {
  });
 
 // configure
-app.configure(feathersSwagger({
-    docsPath:'/docs',
-    version: pkg.version,
-    basePath: '/api',
-    resourcePath: '/example',
-    info: {
-        'title': pkg.name,
-        'description': pkg.description,
-        //'termsOfServiceUrl': 'http://helloreverb.com/terms/',
-        'contact': 'glavin.wiechert@gmail.com',
-        'license': 'MIT',
-        'licenseUrl': 'https://github.com/Glavin001/feathers-swagger/blob/master/LICENSE'
+app
+    // .configure(rest())
+    .use(bodyParser.json())
+    .use(bodyParser.urlencoded({ extended: true }))
+    .configure(feathersSwagger({
+        docsPath:'/docs',
+        version: pkg.version,
+        basePath: '/api',
+        resourcePath: '/example',
+        info: {
+            'title': pkg.name,
+            'description': pkg.description,
+            //'termsOfServiceUrl': 'http://helloreverb.com/terms/',
+            'contact': 'glavin.wiechert@gmail.com',
+            'license': 'MIT',
+            'licenseUrl': 'https://github.com/Glavin001/feathers-swagger/blob/master/LICENSE'
```

Comment:
We should update the licenseUrl to `https://github.com/feathersjs/feathers-swagger/blob/master/LICENSE